src/kakarot/constants.cairo

```diff
@@ -62,8 +62,10 @@ namespace Constants {
     // PROXY
     const INITIALIZE_SELECTOR = 0x79dc0da7c54b95f10aa182ad0a46400db63156920adb65eca2654c0945a463;
     const CONTRACT_ADDRESS_PREFIX = 'STARKNET_CONTRACT_ADDRESS';
-    const EOA_VERSION = 0x11F2231B9D464344B81D536A50553D6281A9FA0FA37EF9AC2B9E076729AEFAA;  // pedersen("KAKAROT_AA_V0.0.1")
-    const CA_VERSION = 10;
+    // hex(pedersen_hash(69796595864846484649,, 0)) ASCII decomposition of EOA_V0.0.1
```

Comment:
this comment is misleading. Actually:

- `EOA_V0.0.1` => `[69, 79, 65, 95, 86, 48, 46, 48, 46, 49]` => `0x454f415f56302e302e31` = `327305290678328760479281`

don't know what you computed actually

Reply:
I have concatenated `[69, 79, 65, 95, 86, 48, 46, 48, 46, 49]` => `69796595864846484649` and hashed with pedersen hash. But I can make it this way also if you think it's better

Reply:
yes plz, while a hex string is an array of bytes, there's no standard way of going from concatenate array of int to bytes